index.html

```diff
@@ -29,16 +30,14 @@
       
       <div class="primary-headline">
             <h1>Hey there. I'm Hiwot<span class="h1color"><br>
-             I'm software developer</span>
+             I'm a software developer</span>
             </h1>
        
         
             <p class="supporting-text">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.
             </p>
         
-
-        <div class="social">
         <ul class="socialt">
             <li>  <img src="image/twitter.svg" /> &nbsp;</li>
             <li> <img src="image/vector.png" /> </li>
```

Comment:
- [ ] As requested by the previous reviewer, Kindly consider wrapping the contents of your list items in an `<a>` tag. This is so that you can easily link to your social media with the `href ` attribute. Your list should look like;
```html
<ul>
  <li>
    <a href="#">
      <img src="#" />
    </a>
  </li>
...
</ul>```

Reply:
As both reviewers mentioned, it's recommended to wrap each icon in an anchor tag so that a usual user can use the icons to connect with you on various social platforms. :100: 
You can find more info about this subject at this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). :+1: 

- [ ] Please consider wrapping each icon from the links in an `a` tag.

Reply:
As both reviewers mentioned, it's recommended to wrap each icon in an anchor tag so that a usual user can use the icons to connect with you on various social platforms. :100: 
You can find more info about this subject at this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). :+1: 

- [ ] Please consider wrapping each icon from the links in an `a` tag.